### PR TITLE
Implemented db-communation for users and user-in-my-group plus API

### PR DESCRIPTION
**Pull request by crishs**
all users and users-in-my-group darabase communication plus API

### CODE REVIEW

File: src/main/java/de/hsh/capstoneris/bo/GroupBO.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package de.hsh.capstoneris.bo;
+import de.hsh.capstoneris.dao.GroupDAO;
```
Comment (SimonOyen) [inline src/main/java/de/hsh/capstoneris/bo/GroupBO.java:2]: GroupDAO class is missing in this pull request, that's why the CI-check fails. Please push it to the same branch

File: pom.xml [hunk 2]
```diff
@@ -49,6 +49,13 @@
             <artifactId>postgresql</artifactId>
             <version>42.2.1</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.4.14.Final</version>
+        </dependency>
+
```
Comment (SimonOyen) [inline pom.xml]: You decided not to use hibernate right? Could you remove the dependency please?

File: src/main/java/de/hsh/capstoneris/dao/UsersDAO.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
+package de.hsh.capstoneris.dao;
+
+import de.hsh.capstoneris.sql.Connection;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+//@Entity
```
Comment (SimonOyen) [inline src/main/java/de/hsh/capstoneris/dao/UsersDAO.java]: Could you also remove the Hibernate-annotations please?

File: src/main/java/persistence.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            <!--<property name="hibernate.cache.region.factory_class"-->
+            <!--value="org.hibernate.cache.ehcache.EhCacheRegionFactory"/>-->
+            <!--<property name="hibernate.javax.cache.provider" value="org.ehcache.jsr107.EhcacheCachingProvider"/>-->
+            <!--&lt;!&ndash;use second level cache&ndash;&gt;-->
+            <!--<property name="hibernate.cache.use_second_level_cache" value="true"/>-->
+            <!--&lt;!&ndash;use 3rd level cache&ndash;&gt;-->
+            <!--<property name="hibernate.cache.use_query_cache" value="true"/>-->
+        </properties>
+    </persistence-unit>
+</persistence>
```
Comment (SimonOyen) [inline src/main/java/persistence.xml]: Please remove the persistence.xml file too

**Review by SimonOyen — CHANGES_REQUESTED** [review]
DB-stuff is good for now, but please remove remaining hibernate-files and lines and add the GroupDAO class

**Comment by crishs** [comment]
requested changes haven been made plus some cleaning

**Pull request merged**